### DISCUSSION TITLE
Be able to select which languages are in the syntax highlighter

### DISCUSSION THREAD

**Discussion by Minipada** [outline/outline#2070] [Ideas]
When writing code in the wiki, there is an option to select the syntax highlighter language.
Right now, the list of languages is static, and I believe each company uses different ones, so it would be good to narrow down the list with what we need. It could be adding in the settings a list of desired languages for example since listing all of them would be too much.

Thanks!

**Comment by tommoor** [comment]
Hi @Minipada I don't think it's necessary to narrow the list down, the current high level plan for this feature is the following:

- Enable code languages to load asyncronously
- Show many more available languages rather than the current small selection
- Include a filter/search field in the UI
- Remember your previous language choice for next time
